Formula/r/readsb.rb

```diff
@@ -0,0 +1,55 @@
+class Readsb < Formula
+  desc "ADS-B decoder swiss knife"
+  homepage "https://github.com/wiedehopf/readsb"
+  url "https://github.com/wiedehopf/readsb/archive/refs/tags/v3.14.1666.tar.gz"
+  sha256 "395063fdc4ba70af14b5a782c230b94dd2bc7c19a6922c935ea8465ddaddc717"
+  license "GPL-3.0-or-later"
+
+  depends_on "pkgconf" => :build
+  depends_on "librtlsdr"
+  depends_on "zstd"
+
+  uses_from_macos "ncurses"
+
+  on_macos do
+    depends_on "libusb"
+  end
+
+  def install
+    ENV["RTLSDR"] = "yes"
+    system "make"
+    bin.install "readsb"
+  end
+
+  service do
+    run [opt_bin/"readsb", "--no-interactive", "--net"]
```

Comment:
This is not a good service command line, that would probably be something that writes json to a directory instead.
And you would use --quiet instead of --no-interactive.

Otherwise might want to open some ports as by default --net does not open any (in contrast to most dump1090 variants)

But really i have no clue what this service is meant to do.

Reply:
I'll remove the service. It seems like something end-users should setup.